public/app/plugins/panel/debug/panelcfg.gen.ts

```diff
@@ -0,0 +1,36 @@
+// Code generated - EDITING IS FUTILE. DO NOT EDIT.
+//
+// Generated by:
+//     public/app/plugins/gen.go
+// Using jennies:
+//     TSTypesJenny
+//     PluginTSTypesJenny
+//
+// Run 'make gen-cue' from repository root to regenerate.
+
+export const PanelCfgModelVersion = Object.freeze([0, 0]);
+
+export type UpdateConfig = {
```

Comment:
@sdboyer we weren't quite sure how to approach this. The handwritten type definition looked like:

```
export type UpdateConfig = {
  [K in keyof UpdateCounters]: boolean;
};
```

Is there any support for `keyof` in cuetsy?

Reply:
It's definitely possible to construct the equivalent rule in CUE, but it'd be pretty weird to teach cuetsy how to actually produce that output.

There's probably a happy medium here though where you put the expected logic into the CUE type, then use a handwritten veneer in TS to get what you want there

Reply:
i pushed a commit that turns this into a derived thing